pkg/kritis/review/validating_transport.go

```diff
@@ -101,51 +102,80 @@ func (avt *AttestorValidatingTransport) validatePublicKeyId(pubKey v1beta1.Publi
 	return nil
 }
 
-func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string) ([]attestation.ValidatedAttestation, error) {
-	keys := map[string]v1beta1.PublicKey{}
-	numKeys := len(avt.Attestor.Spec.PublicKeys)
-	for i, pubKey := range avt.Attestor.Spec.PublicKeys {
-		if err := avt.validatePublicKey(pubKey); err != nil {
+func (avt *AttestorValidatingTransport) parsePublicKeys() []cryptolib.PublicKey {
+	attestorKeys := avt.Attestor.Spec.PublicKeys
+	numKeys := len(attestorKeys)
+	publicKeys := []cryptolib.PublicKey{}
+
+	for i, attestorKey := range attestorKeys {
+		if err := avt.validatePublicKey(attestorKey); err != nil {
```

Comment:
I might miss something, should we check keytype here?

Reply:
Done, added. Perhaps we can remove this check once PKIX verification is implemented.

Reply:
Sure. I'm bringing up `check` mostly because we call `attestorKey.AsciiArmoredPgpPublicKey` later in this function, not sure why I left the comment here...